package.json

```diff
@@ -39,14 +39,14 @@
 		"object-fit-videos": "^1.0.3",
 		"ophan-tracker-js": "1.4.0",
 		"preact": "^10.5.13",
-		"prebid.js": "https://github.com/guardian/Prebid.js#0e9f87a",
+		"prebid.js": "7.15.0",
```

Comment:
This cannot be right. We have our own custom build of Prebid. cc. @guardian/commercial-dev